iOS Client/Assets.xcassets/pause-bars.imageset/Contents.json

```diff
@@ -2,17 +2,15 @@
   "images" : [
     {
       "idiom" : "universal",
-      "filename" : "ic_pause.png",
+      "filename" : "pause-bars.png",
       "scale" : "1x"
     },
     {
       "idiom" : "universal",
```

Comment:
interesting, are we just rolling with one image for now?

Reply:
The other image was too small and getting warped at large sizes so just found a new image. we can get some larger scales later if necessary